src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -395,6 +395,32 @@
               ]
             }
           ]
+        },
+        {
+            "path": "/v1/debug/bundle/actions/create",
```

Comment:
i'm not sure i understand the url structure. `post` is the verb, so why does the url need a verb? contrast this with posting to `/v1/debug/bundle` to achieve the same result.